.github/CODEOWNERS

```diff
@@ -106,17 +112,22 @@
 # PRLabel: %Communication - Common
 /sdk/communication/**/_shared/                                       @Azure/acs-identity-sdk @AikoBB @maximrytych-ms @mjafferi-msft
 
+# ServiceOwners: @lynshi
+# ServiceLabel: %Confidential Ledger
 # PRLabel: %Confidential Ledger
-/sdk/confidentialledger/azure-confidentialledger/                    @PallabPaul
+/sdk/confidentialledger/azure-confidentialledger/                    @PallabPaul @lynshi
 
+# ServiceOwners: @mengaims
+# ServiceLabel: %Content Safety
 # PRLabel: %Content Safety
 /sdk/contentsafety/                                                  @mengaims @JieZhou000
 
 # AzureSDKOwners: @mccoyp
 # ServiceLabel: %KeyVault
 # PRLabel: %KeyVault
-/sdk/keyvault/                                                       @schaabs @mccoyp
+/sdk/keyvault/                                                       @mccoyp
 
+# ServiceLabel: %Load Testing
```

Comment:
This form will assume the service owns the library, but will include Izzy for all "Service Attention" labels that get applied to "Load Testing" issues. 

You may want to split the ServiceLabel + ServiceOwners out into a new block and let Izzy just stay on the PR path.

```text
# PRLabel: %Load Testing
/sdk/loadtesting/azure-developer-loadtesting/                        @msyyc @iscai-msft

# ServiceLabel %Load Testing
# ServiceOwners: @msyyc
```